drivers/gles3/storage/material_storage.cpp

```diff
@@ -1561,8 +1563,8 @@ MaterialStorage::MaterialStorage() {
 		actions.renames["MODEL_NORMAL_MATRIX"] = "model_normal_matrix";
 		actions.renames["VIEW_MATRIX"] = "scene_data.view_matrix";
 		actions.renames["INV_VIEW_MATRIX"] = "scene_data.inv_view_matrix";
-		actions.renames["PROJECTION_MATRIX"] = "projection_matrix";
-		actions.renames["INV_PROJECTION_MATRIX"] = "inv_projection_matrix";
+		actions.renames["PROJECTION_MATRIX"] = "scene_data.projection_matrix";
```

Comment:
This looks wrong, `projection_matrix` and `inv_projection_matrix` are calculated in the shader as they are assigned differently for multiview. This change will break multiview.